examples/udt/udt.c

```diff
@@ -248,20 +274,20 @@ int main(int argc, char* argv[]) {
     return -1;
   }
 
-  schema_meta = cass_session_get_schema_meta(session);
+  execute_schema_query(session,
+                       "CREATE KEYSPACE examples WITH replication = { \
+                       'class': 'SimpleStrategy', 'replication_factor': '3' }");
 
-  execute_query(session,
-                "CREATE KEYSPACE examples WITH replication = { \
-                           'class': 'SimpleStrategy', 'replication_factor': '3' }");
+  execute_schema_query(session,
+                       "CREATE TYPE examples.phone_numbers (phone1 int, phone2 int)");
 
-  execute_query(session,
-                "CREATE TYPE examples.phone_numbers (phone1 int, phone2 int)");
+  execute_schema_query(session,
+                       "CREATE TYPE examples.address (street text, city text, zip int, phone set<frozen<phone_numbers>>)");
 
-  execute_query(session,
-                "CREATE TYPE examples.address (street text, city text, zip int, phone set<frozen<phone_numbers>>)");
+  execute_schema_query(session,
+                       "CREATE TABLE examples.udt (id timeuuid, address frozen<address>, PRIMARY KEY(id))");
 
-  execute_query(session,
-                "CREATE TABLE examples.udt (id timeuuid, address frozen<address>, PRIMARY KEY(id))");
+  schema_meta = cass_session_get_schema_meta(session);
```

Comment:
I hope adding this one line will update the schema meta, as it is not required to update for each schema change in this scenario.

Reply:
Yes, you're correct.